src/movement/test/test_thruster_sabertooth.cpp

```diff
@@ -0,0 +1,120 @@
+#include <gtest/gtest.h>
+#include "ros/ros.h"
+#include "robosub/thruster.h"
+#include "utility/serial_testbench.hpp"
+#include "utility/serial.hpp"
+
+typedef struct thruster_info
```

Comment:
This struct is a copy from [thruster_sabertooth.cpp](https://github.com/PalouseRobosub/robosub/blob/dev/src/movement/thruster_sabertooth.cpp) file. It would be more suitable to put this into a header file.